scripts/build_azuresdk.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash -e
+#
+# build azure-cpp-lite
+#
+function usage() {
+    echo "Usage: `basename $0` [-t <Release|Debug>]"
+    echo "Build AZURE SDK"
+    echo "-t <Release/Debug> : Release or Debug builds"
+    exit 2
+}
+set -o pipefail
+
+export CC="/usr/lib64/ccache/gcc52 -g"
+export CXX="/usr/lib64/ccache/g++52 -g"
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $DIR/_init.sh $@
+AZURE_SOURCE_DIR=$DEPS_DIR/azure-storage-cpplite
+AZURE_BUILD_DIR=$DEPENDENCY_DIR/azure
+AZURE_CMAKE_BUILD_DIR=$AZURE_SOURCE_DIR/cmake-build
+
+GIT_REPO="https://github.com/snowflakedb/azure-storage-cpplite.git"
+CLONE_CMD="git clone -b master $GIT_REPO $AZURE_SOURCE_DIR"
+VERSION="v0.1.1"
+
+if [ ! -d $AZURE_SOURCE_DIR ]; then
+  n=0 
+  # retry 5 times on cloning
+  until [ $n -ge 5 ] 
+  do
+    if $CLONE_CMD ; then
+      break
+    fi  
+    n=$[$n+1]
+  done
+  
+  if [ ! -d $AZURE_SOURCE_DIR ]; then
+    echo "[Error] failed to clone repo from $GIT_REPO"
+    exit 1
+  fi  
+
+  cd $AZURE_SOURCE_DIR
+  git checkout tags/$VERSION -b $VERSION
+else
+  cd $AZURE_SOURCE_DIR
+  git fetch
+  git checkout tags/$VERSION -b $VERSION || true
+fi
+
+
+azure_configure_opts=()
+if [[ "$target" != "Release" ]]; then
+    azure_configure_opts+=("-DCMAKE_BUILD_TYPE=Debug")
+    AZURE_CMAKE_BUILD_DIR=$AZURE_SOURCE_DIR/cmake-build-debug
+else
+    azure_configure_opts+=("-DCMAKE_BUILD_TYPE=Release")
+fi
+azure_configure_opts+=(
+    "-DCMAKE_VERBOSE_MAKEFILE:BOOL=OFF"
+    "-DCMAKE_C_COMPILER=$GCC"
+    "-DCMAKE_CXX_COMPILER=$GXX"
+    "-DCMAKE_INSTALL_PREFIX=$AZURE_BUILD_DIR"
+    "-DBUILD_SHARED_LIBS=OFF"
+    "-DUSE_OPENSSL=true"
+    "-DOPENSSL_VERSION_NUMBER=0x11100000L"
```

Comment:
@rvishureddy , Better moving this version to the top of this file or config. Related to @notkriswagner question; does this matter to Azure SDK build? I mean without the version number, what will happen? Ideally version neutral build scrip would be great for maintenance.

Reply:
Openssl API's changed with version 1.1.1. libSnowflakeclient and ODBC both use openssl 1.1.1b and to have corresponding API's invoked in Azure sdk it should be compiled with the above flag.

Reply:
Ok, thanks for confirmation.